### PR TITLE
Add Button to copy Lobby Link

### PR DESCRIPTION
**Pull request by wtachau**
https://trello.com/c/NwI9t8lG/6-m-l-improve-host-join-experience

I tried to avoid using an extra library, but I couldn't figure out how to get it working using the `inputRef` of the TextField, programmatically selecting that text (though I could get the input to focus), then calling `document.execCommand('copy')`.

Play_the_Fishbowl_Game__aka_Salad_Bowl__-_Online___Free

Play_the_Fishbowl_Game__aka_Salad_Bowl__-_Online___Free

### CODE REVIEW

**Comment by avimoondra** [comment]
[[M-L] Improve Host/Join Experience](https://trello.com/c/NwI9t8lG/6-m-l-improve-host-join-experience)

File: app/src/pages/Lobby/index.tsx [hunk 1]
```diff
@@ -36,9 +38,56 @@ const useStyles = makeStyles((theme: Theme) =>
...
+
+  React.useEffect(() => {
+    let timeout: NodeJS.Timeout
+    if (copyButtonClicked) {
+      timeout = setTimeout(() => {
+        setCopyButtonClicked(false)
+      }, 1000)
+    }
+    return () => timeout && clearTimeout(timeout)
+  }, [copyButtonClicked])
```
Comment (avimoondra) [inline app/src/pages/Lobby/index.tsx:51]: added a little flourish here, cc: @wtachau

File: app/src/pages/Lobby/index.tsx [hunk 2]
```diff
@@ -36,9 +38,56 @@ const useStyles = makeStyles((theme: Theme) =>
...
+
   return (
     <Grid item>
-      {`Share the code with everyone playing`}
+      {`Share your link with everyone playing`}
+      <Grid container spacing={2} style={{ paddingTop: 8, paddingBottom: 8 }}>
```
Comment (avimoondra) [inline app/src/pages/Lobby/index.tsx:56]: for future - one solution here to stick w/the framework is wrappign w/ `<Box pt={1} pb={1}>`

cc: @wtachau

File: app/src/pages/Lobby/index.tsx (hunk 1 above)
Comment (wtachau) [reply app/src/pages/Lobby/index.tsx:51]: nice 👍

**Pull request merged**